go/cli/commands/build.go

```diff
@@ -114,14 +155,74 @@ func (r *BuildRunner) dockerfileExist() bool {
 	return true
 }
 
-func (r *BuildRunner) createDockerfile() error {
-	dockerfileContent, err := f.ReadFile(builtinDockerfilePath)
+func (r *DockerBuilder) createDockerfile() error {
+	dockerfileContent, err := f.ReadFile(BuiltinDockerfilePath)
+	if err != nil {
+		return err
+	}
+	if err = os.WriteFile(DockerfilePath, dockerfileContent, 0644); err != nil {
+		return err
+	}
+	fmt.Println("created Dockerfile")
+	return nil
+}
+
+func (r *KoBuilder) GuaranteeKoInstalled() error {
+	_, err := execLookPathFn("ko")
+	if err == nil {
+		return nil
+	}
+	if _, err = execCmdFn(nil, "go", "install", "github.com/google/ko@latest"); err != nil {
+		return err
+	}
+	fmt.Println("successfully installed ko")
+	return nil
+}
+func (r *KoBuilder) Build() error {
+	args := []string{"build", "-B", "--tags", r.Tag}
+	envs := []string{KoDockerRepoEnvVar + "=" + r.Repo}
+	out, err := execCmdFn(envs, "ko", args...)
 	if err != nil {
 		return err
 	}
-	if err := os.WriteFile(DockerfilePath, dockerfileContent, 0644); err != nil {
+
+	if r.Repo == KoLocalRepo {
+		color.Green("Image builds successfully. Now you can publish the image %v", out)
```

Comment:
Nit: "Image built successfully.  Now you ..."

Reply:
Thanks! updated here and elsewhere